docs/middlewares.md

```diff
@@ -140,6 +141,104 @@ handler(event, context, (_, response) => {
 })
 ```
 
+## [functionShield](/src/middlewares/functionShield.js)
+
+Hardens AWS Lambda execution environment:
+* By monitoring (or blocking) outbound network traffic from your function, you can be certain that your data is never leaked
```

Comment:
`outbound network traffic to public internet` to be specific? or mention that traffic to AWS services are not affected, that'd be my first question when I see this sentence